server/py/framework/db/sqldb/models.py

```diff
@@ -862,7 +862,7 @@ class AlertActivation(Base, mlrun.utils.db.BaseModel):
         )
 
         id = Column(Integer, autoincrement=True)
-        activation_time = Column(SQLTypesUtil.datetime(), nullable=False)
+        activation_time = Column(SQLTypesUtil.datetime(fsp=3), nullable=False)
```

Comment:
Please add comment that it should not be changed for future :) (and same for reset_time)